docs/sources/next/extensions/explore.md

```diff
@@ -135,14 +119,6 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
         <h4>xk6-git</h4>
         <p>Clone Git repositories from tests</p>
     </a>
-    <a href="https://github.com/AckeeCZ/xk6-google-iap" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
The extension does not meet the registry requirements
https://github.com/AckeeCZ/xk6-google-iap/issues/10
https://github.com/AckeeCZ/xk6-google-iap/issues/9
https://github.com/AckeeCZ/xk6-google-iap/issues/8